pkg/ccl/changefeedccl/kvfeed/kv_feed_test.go

```diff
@@ -173,8 +177,11 @@ func TestKVFeed(t *testing.T) {
 		// Assert that number of events emitted from the kvfeed matches what we
 		// specified in the testcase.
 		testG.GoCtx(func(ctx context.Context) error {
-			for events := 0; events < tc.expEvents; events++ {
-				_, err := buf.Get(ctx)
+			for eventIdx := 0; eventIdx < tc.expEventsCount; eventIdx++ {
+				e, err := buf.Get(ctx)
+				if tc.expEvents != nil {
```

Comment:
nit: this if branch should go under the `assert.NoError`

Reply:
Done.